integration/appaccess/jwt.go

```diff
@@ -52,4 +53,12 @@ func verifyJWT(t *testing.T, pack *Pack, token, appURI string) {
 	require.NoError(t, err)
 	require.Equal(t, pack.username, claims.Username)
 	require.Equal(t, pack.user.GetRoles(), claims.Roles)
+
+	filteredTraits := wrappers.Traits{}
+	for trait, values := range pack.user.GetTraits() {
+		if values != nil {
```

Comment:
I'd check `len(values) > 0` instead of nil, this will handle both nil and non-nil but empty.

Reply:
Sounds good.